native_client/javascript/deepspeech.i

```diff
@@ -62,28 +62,25 @@ using namespace node;
 
 %nodefaultctor ModelState;
 %nodefaultdtor ModelState;
-%nodefaultdtor Metadata;
-%nodefaultctor Metadata;
-%nodefaultctor MetadataItem;
-%nodefaultdtor MetadataItem;
 
-%extend Metadata {
-  v8::Local<v8::Value> items;
-  v8::Local<v8::Value> items_get() {
-    v8::Local<v8::Value> jsresult = SWIGV8_ARRAY_NEW();
-    for (int i = 0; i < self->num_items; ++i) {
-      jsresult = SWIGV8_AppendOutput(jsresult, SWIG_NewPointerObj(SWIG_as_voidptr(&self->items[i]), SWIGTYPE_p_MetadataItem, SWIG_POINTER_OWN));
-    }
-  fail:
-    return jsresult;
+%typemap(out) MetadataItem* %{
+  $result = SWIGV8_ARRAY_NEW();
+  for (int i = 0; i < arg1->num_items; ++i) {
+    SWIGV8_AppendOutput($result, SWIG_NewPointerObj(SWIG_as_voidptr(&result[i]), SWIGTYPE_p_MetadataItem, 0));
```

Comment:
This is not perfect, but since it looks like anyway V8 has no guarantee to call the destructor, and this is breaking in some weird way in our case, maybe we can live with that ?